mail/postfix/src/opnsense/mvc/app/models/OPNsense/Postfix/General.xml

```diff
@@ -47,6 +47,11 @@
             <default>51200000</default>
             <Required>Y</Required>
         </message_size_limit>
+        <masquerade_domains type="CSVListField">
+            <Required>N</Required>
+            <mask>/^([0-9a-zA-Z\.\,\-\_]){1,128}$/u</mask>
```

Comment:
the regex may be improved

Reply:
Comma cause of CSVListField. Otherwise more than one entra is invalid

Reply:
Yep, looks good, thanks :)